fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/type/LocationParmBehaviorUtil.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * (C) Copyright IBM Corp. 2019
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.persistence.jdbc.util.type;
+
+import static com.ibm.fhir.persistence.jdbc.JDBCConstants.BIND_VAR;
+import static com.ibm.fhir.persistence.jdbc.JDBCConstants.DOT;
+import static com.ibm.fhir.persistence.jdbc.JDBCConstants.LATITUDE_VALUE;
+import static com.ibm.fhir.persistence.jdbc.JDBCConstants.LEFT_PAREN;
+import static com.ibm.fhir.persistence.jdbc.JDBCConstants.LONGITUDE_VALUE;
+import static com.ibm.fhir.persistence.jdbc.JDBCConstants.RIGHT_PAREN;
+import static com.ibm.fhir.persistence.jdbc.util.QuerySegmentAggregator.PARAMETER_TABLE_ALIAS;
+
+import java.util.List;
+
+import com.ibm.fhir.persistence.jdbc.JDBCConstants;
+import com.ibm.fhir.persistence.jdbc.JDBCConstants.JDBCOperator;
+import com.ibm.fhir.search.location.bounding.Bounding;
+import com.ibm.fhir.search.location.bounding.BoundingBox;
+import com.ibm.fhir.search.location.bounding.BoundingRadius;
+
+/**
+ * Location Behavior Util generates SQL and loads the variables into bind
+ * variables.
+ * <br>
+ * The bounding areas at this point are already valid as the fhir-search
+ * component has processed the received point into a set of bounding boxes.
```

Comment:
is this right?  i see two buildLocationSearchQuery methods: one takes a List of Boundings (not boxes) and the other takes a `single` bounding box.

Reply:
There are actually three methods
1 - coordinates the type of the boundingarea 
2 - for the radius 
3 - for the box

Reply:
ok, i guess my point was that the "the fhir-search component has processed the received point into a set of bounding boxes" comment doesn't see to belong here (at the class level)...maybe move it to one of the methods or just delete it?